### DISCUSSION TITLE
mixdrop failed to download

### DISCUSSION THREAD

**Discussion by meatyguitar** [aclap-dev/video-downloadhelper#586] [Help and Q&A]
### Your question

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [in#0 @ 0000014787f574c0] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://s-delivery45.mxcontent.net/v/f1cb8f17919fd407ea441993e4700491.mp4?s=ChX1TCQDgUAHGm6rEt0W5Q&e=1716516076&_t=1716502909.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "id": "coapp_failure_e38963db-f178-4792-ac47-955e448d7e93",
    "downloadable_id": "downloadable_5909462355433962"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:125.0) Gecko/20100101 Firefox/125.0
{
    "download_directory": "D:\\[2024 DL]",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 10
}

**Comment by paulrouget** [comment]
triage: ff-failure 403
